packages/ws-server/src/webSocketServer.ts

```diff
@@ -62,7 +63,13 @@ app.ws.use((ctx) => {
     ctx.websocket.id = relay.subs()?.generateId();
     logger.info(`${LOGGER_PREFIX} new connection ${ctx.websocket.id}`);
 
-    connectedClients = ctx.app.server._connections;
+    // Limit connection TTL and close connection if its reached
+    setTimeout(() => {
+        if (ctx.websocket.readyState !== 3) { // 3 = CLOSED, Avoid closing already closed connections
+            logger.info(`${LOGGER_PREFIX} closing connection ${ctx.websocket.id} due to reaching TTL of ${MAX_CONNECTION_TTL}ms`);
+            ctx.websocket.close();
```

Comment:
should we have a try catch around the `close()` to catch any exceptions?

Reply:
✅ Done